setup.cfg

```diff
@@ -0,0 +1,90 @@
+[metadata]
+description = Industrial-strength Natural Language Processing (NLP) in Python
+url = https://spacy.io
+author = Explosion
+author_email = contact@explosion.ai
+license = MIT
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Cython
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+
+[options]
+zip_safe = false
+include_package_data = true
+scripts =
+    bin/spacy
+python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
+setup_requires =
+    wheel
+install_requires =
+    numpy>=1.15.0
+    murmurhash>=0.28.0,<1.1.0
+    cymem>=2.0.2,<2.1.0
+    preshed>=3.0.2,<3.1.0
+    thinc>=7.1.1,<7.2.0
+    blis>=0.4.0,<0.5.0
+    plac<1.0.0,>=0.9.6
+    requests>=2.13.0,<3.0.0
+    wasabi>=0.2.0,<1.1.0
+    srsly>=0.1.0,<1.1.0
+    pathlib==1.0.1; python_version < "3.4"
+
+[options.extras_require]
+cuda =
+    thinc_gpu_ops>=0.0.1,<0.1.0
+    cupy>=5.0.0b4
+cuda80 =
+    thinc_gpu_ops>=0.0.1,<0.1.0
+    cupy-cuda80>=5.0.0b4
+cuda90 =
+    thinc_gpu_ops>=0.0.1,<0.1.0
+    cupy-cuda90>=5.0.0b4
+cuda91 =
+    thinc_gpu_ops>=0.0.1,<0.1.0
+    cupy-cuda91>=5.0.0b4
+cuda92 =
+    thinc_gpu_ops>=0.0.1,<0.1.0
+    cupy-cuda92>=5.0.0b4
+cuda100 =
+    thinc_gpu_ops>=0.0.1,<0.1.0
+    cupy-cuda100>=5.0.0b4
+# Language tokenizers with external dependencies
+ja =
+    mecab-python3==0.7
+ko =
+    natto-py==0.9.0
+th =
+    pythainlp>=2.0
+
+[bdist_wheel]
+universal = true
```

Comment:
This is wrong, right? I'm pretty sure we don't provide universal wheels

Reply:
Ah yeah, I guess. Sorry, just copy-pasted that.